### PR TITLE
Add moving rudder to F/CTL ECAM page

### LINKED ISSUE #199: Need to enable rudder in F-CTL screen
Hey gang!  The rudder movement needs to be added to the F-CTL screen.  All other controls are working.

Thanks.  Dennis

### PR DESCRIPTION
**Pull request by lukecologne**
Fixes #199

**Summary of Changes**
* Adds functionality to the F/CTL rudder indicator.

**Screenshots (if necessary)**
grafik

### CODE REVIEW

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_FTCL.js [hunk 1]
```diff
@@ -88,6 +91,11 @@ var A320_Neo_LowerECAM_FTCL;
             this.leftElevatorCursor.setAttribute("d", leCursorPath);
             this.rightElevatorCursor.setAttribute("d", reCursorPath);
 
+            // Update rudder
+            var rudderDeflectPct = SimVar.GetSimVarValue("RUDDER DEFLECTION PCT", "percent over 100");
+            let rudderAngle = -rudderDeflectPct * 29.06;
```
Comment (Benjozork) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_FTCL.js:96]: Both can be `const`.

**Comment by lukecologne** [comment]
I would also like to add the rudder limits indicator, but I had no luck finding any way of getting the current limit, although it definetely exists. So if anyone has any ideas, I'd be glad to hear them.

**Review by AdenFlorian — APPROVED** [review]
tested and LGTM!

**Comment by JackZ974** [comment]
> 
> 
> I would also like to add the rudder limits indicator, but I had no luck finding any way of getting the current limit, although it definetely exists. So if anyone has any ideas, I'd be glad to hear them.

The rudder travel limits vary with speed. The TLU start reducing from 25° above 160kts up to 3.4° at VMO(380 kts), but it's not alinear function, more a logarithmic one.

**Comment by lukecologne** [comment]
That's what I suspected, but since I didn't have any actual numbers I couldn't do anything. Is that that the actual curve, or is it just a rough approximation?

**Comment by JackZ974** [comment]
FCOM extract

**Comment by lukecologne** [comment]
Cool, thanks, I'll try it out

**Comment by JackZ974** [comment]
The ideal would be once this value is computed, to be transmitted to the FBW system so that the rudder physical travel would be limited as well. 
Note that there is an aural alert " STOP RUDDER INPUT3 and RED PFD message with master warning when excessive amount of rudder inputsvare detected (can be very dangerous at high speed).
 At slats extension, full rudder travel limit is restored.

**Comment by lukecologne** [comment]
The sim seems to already limit rudder travel at high speeds, that's why I asked if there was some way to get the limit programmatically. Hopefully it matches, but I suspect it won't

**Comment by JackZ974** [comment]
Apparently Asobo kinda implemented some flight controls limits, but forgot some as well. 
In manual flight, I can actually bank to the right over 90° and end up completely inverted, but if I bank to the left it won't allow me go past 67degrees (as it should be in normal law). Go figure...

**Pull request merged**